java/client/src/main/java/glide/api/models/BaseTransaction.java

```diff
@@ -5623,6 +5624,23 @@ public T hscan(@NonNull String key, @NonNull String cursor, @NonNull HScanOption
         return getThis();
     }
 
+    /**
+     * Blocks the current client until all the previous write commands are successfully transferred
+     * and acknowledged by at least <code>numreplicas</code> of replicas. If <code>timeout</code> is
+     * reached, the command returns even if the specified number of replicas were not yet reached.
+     *
+     * @param numreplicas The number of replicas to reach.
+     * @param timeout The timeout value specified in milliseconds. A value of <code>0</code> will
```

Comment:
This should not block if you are using it in a transaction. Please update the param comment for this and the general comment above.